src/containers/App.js

```diff
@@ -338,8 +338,12 @@ class App extends Component {
     }
 
     if (
-      prevProps.accountStatus === Status.LOADING &&
-      this.props.accountStatus !== Status.LOADING
+      (prevProps.accountStatus === Status.LOADING &&
+        this.props.accountStatus !== Status.LOADING) ||
+      matchPath(getPathname(this.props.location), {
```

Comment:
I don't quiet understand why this is needed

Reply:
I should add a comment. Essentially, the `isReady` flag gets sent into `AppRedirectPopover`'s `isEnabled` flag. Presumably this was used to wait until the rest of the UI was ready before loading the redirect popover. However, the `isReady` never gets set to true if there's no user to fetch, so the popover never shows on pages where there is no user (like sign-in). In theory, this could be a check for simply "is this route an auth route" but kept it narrow for the sign up referrer case.